infrastructures/openshift/src/main/java/org/eclipse/che/workspace/infrastructure/openshift/project/OpenShiftPods.java

```diff
@@ -156,6 +165,48 @@ public void onClose(KubernetesClientException cause) {
   }
 
   /**
+   * Starts watching the pods inside OpenShift namespace and registers a specified handler for such
+   * events. Note that watcher can be started only once so two times invocation of this method will
+   * not produce new watcher and just register the event handlers.
+   *
+   * @param handler pod action events handler
+   * @throws InfrastructureException if any error occurs while watcher starting
+   */
+  public void watch(PodActionHandler handler) throws InfrastructureException {
```

Comment:
Looks like tests are not modified in accordance with this change

Reply:
`OpenShiftPods` it's a part of wrapper mechanism that we use for interaction with os client and we do not test this interaction at all.